evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -52,9 +53,8 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
                         </div>
                         {% for answer in result.answers %}
                             <div class="slider-item card-body" data-layer="2" data-id="{{ answer.id }}"{% if contributor %} data-contribution="yes"{% endif %}
-                                {% if answer.is_reviewed %}data-review="{% if answer.is_published %}publish{% elif answer.is_private %}make_private{% else %}hide{% endif %}"{% endif %}
-                            >
-                            <a class="textanswer-link" href="{% url 'staff:evaluation_textanswer_edit' semester.id evaluation.id answer.id %}">{{ answer.answer|linebreaksbr }}</a>
+                                {% if answer.is_reviewed %}data-review="{% if answer.is_published %}publish{% elif answer.is_private %}make_private{% else %}hide{% endif %}"{% endif %}>
+                            <p>{{ answer.answer|linebreaksbr }}</p>
```

Comment:
```suggestion
                            {{ answer.answer|linebreaksbr }}
```

Reply:
we don't need to add a tag here

Reply:
please also check the indentation inside the `div`, it currently doesn't seem right